backend/src/utils/web_utils.py

```diff
@@ -12,64 +12,38 @@
 
 engine = PromptEngine()
 
+
 def search_urls(search_query, num_results=10) -> str:
     logger.info(f"Searching the web for: {search_query}")
     urls = []
     try:
         for url in search(search_query, num_results=num_results):
             urls.append(url)
-        return json.dumps({
-            "status": "success",
-            "urls": urls,
-            "error": None
-        })
+        return json.dumps({"status": "success", "urls": urls, "error": None})
     except Exception as e:
         logger.error(f"Error during web search: {e}")
-        return json.dumps({
-            "status": "error",
-            "urls": [],
-            "error": str(e)
-        })
+        return json.dumps({"status": "error", "urls": [], "error": str(e)})
 
 
 def scrape_content(url, limit=100000) -> str:
     try:
         logger.info(f"Scraping content from URL: {url}")
         response = requests.get(url)
         response.raise_for_status()
-        soup = BeautifulSoup(response.text, 'html.parser')
-        paragraphs = soup.find_all('p')
-        content = ' '.join([para.get_text() for para in paragraphs])
-        return json.dumps({
-            "status": "success",
-            "content": content[:limit],
-            "error": None
-        })
+        soup = BeautifulSoup(response.text, "html.parser")
+        paragraphs = soup.find_all("p")
+        content = " ".join([para.get_text() for para in paragraphs])
+        return json.dumps({"status": "success", "content": content[:limit], "error": None})
     except Exception as e:
         logger.error(f"Error scraping {url}: {e}")
-        return json.dumps({
-            "status": "error",
-            "content": "",
-            "error": str(e)
-        })
+        return json.dumps({"status": "error", "content": "", "error": str(e)})
+
 
 def summarise_content(search_query, contents, llm, model) -> str:
     try:
-        summariser_prompt = engine.load_prompt(
-            "summariser",
-            question=search_query,
-            content=contents
-        )
-        response = llm.chat(model, summariser_prompt, "")
-        return json.dumps({
-            "status": "success",
-            "response": response,
-            "error": None
-        })
+        summariser_prompt = engine.load_prompt("summariser", question=search_query, content=contents)
+        response = llm.chat(model, summariser_prompt, "", return_json=True)
+        return json.dumps({"status": "success", "response": response, "error": None})
```

Comment:
These were automatically flattened by the ruff formatter extension. If we'd rather keep these on separate lines it might be worth investigating the ruff config?

Reply:
I would prefer this split over several lines for readability, especially when the json is a large structure. Could you raise a ticket so we dont lose this investigation/change?

Reply:
Yeah definitely!
I've made this [ticket](https://scottlogic.atlassian.net/browse/IG-137) to track this